### DISCUSSION TITLE
Can't import @Mapper annotation

### DISCUSSION THREAD

**Discussion by serdarakyol** [mapstruct/mapstruct#3167] [Q&A]
I have a spring boot app. You can access the entire app from [here](https://github.com/serdarakyol/spring-boot-app). I have followed the small documentation of [MapStruct](https://mapstruct.org/) but I am not able to import any of MapStruct annotation. I imported MapStruct on the `pom.xml` and then run `mvn clean install` but the problem still exists. I am having `The import org.mapstruct cannot be resolved` error.

Current [pom.xml](https://github.com/serdarakyol/spring-boot-app/blob/d1a1413a606cbd6d382ebea0a0845178b6489f05/pom.xml) file

Can anyone help me with that, please?

**Comment by vld0** [comment]
I do not see mapstruct jar dependency : https://github.com/mapstruct/mapstruct#using-mapstruct

**Reply by serdarakyol** [reply]
Thanks for the feedback. It is included in the PR that I have created but I wasn't able to import that. Then I created a new project and moved my all components to new project. Then the problem was solved. I think it was related to maven, I am not sure about that but now I am able to use that
